src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClients/FailOverSupportedConfigurationClient.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure;
+using Azure.Core;
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Constants;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.ConfigurationClients
+{
+    internal class FailOverSupportedConfigurationClient : IConfigurationClient
+    {
+        private const int HttpStatusCodeRequestThrottled = 429;
```

Comment:
I think I commented it previously. You shouldn't need to hard code 429.

Reply:
`HttpStatusCode.TooManyRequests` is only available after `netstandard2.1`. We use `netstandard2.0`.
So, it is not available to us unless we upgrade. 
 [System.Net.HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netstandard-2.0#system-net-httpstatuscode-multiplechoices)

Reply:
I see. Thanks. I didn't see your response in my previous comment.

Reply:
BTW, do you mind to add a comment to explain why it must be defined here so I don't ask the same question again next time? :)